### PR TITLE
Fixed typos and added a relative link

### PR DESCRIPTION
**Pull request by shivam71**
Corrected spelling mistakes and converted the absolute url to relative one for the LICENSE file so that it doesn't point to the file in the main branch .

### CODE REVIEW

**Review by Achal1607 — APPROVED** [review]
LGTM. Thanks

File: README.md [hunk 1]
```diff
@@ -65,20 +65,20 @@ Project Explorer provides an overview of logical project structure, groups sourc
...
     * Select this configuration, then click the ![Run](vscode/images/run.png) 
     * Select either from available process or enter the port to connect to JVM running with JDWP.
-    * __Attach to Shared Memory__ is available on Windows in addtion to above mentioned _Attach..._
+    * __Attach to Shared Memory__ is available on Windows in addition to above mentioned _Attach..._
 
 Default launch configurations provided by Language Server can modified in `launch.json` file.
 
 ### Run Configurations panel
-Program arguments, VM options, evironment variables,... can be set in Run Configuration panel a part of Explorer. The panel is sufficient for all typical use-cases Java programmer faces. Only advanced, expert scenarios may require touching of `launch.json` (which still takes precedence).  
+Program arguments, VM options, environment variables,... can be set in Run Configuration panel a part of Explorer. The panel is sufficient for all typical use-cases Java programmer faces. Only advanced, expert scenarios may require touching of `launch.json` (which still takes precedence).
```
Comment (sid-srini) [inline README.md:81]:
```suggestion
Program arguments, VM options, environment variables, etc. can be set in the Run Configuration panel within VS Code's Activity bar Explorer view. The panel caters for most typical use-cases that programmers face for running their Java programs. Only advanced, expert scenarios may require using `launch.json` to define customised launch targets which take precedence over the Run Configuration.  
```

File: README.md [hunk 2]
```diff
@@ -65,20 +65,20 @@ Project Explorer provides an overview of logical project structure, groups sourc
 
 ## Debugger and Launch Configurations
 Language Server __Java+ ...__ launch configuration supports debugging and running Java applications using JDK11 or newer.
```
Comment (sid-srini) [inline README.md:67]: change JDK11 to JDK8?

File: README.md [hunk 3]
```diff
@@ -65,20 +65,20 @@ Project Explorer provides an overview of logical project structure, groups sourc
...
+1. The launch configuration (debugger) is invoked when `Run main | Debug main` codelens is selected in the code.
 2. Or __Java+...__ is selected in __Run and Debug__ activity panel.  
 ![Debug configurations](vscode/images/debuggers.png)
 ### Launch Configurations
 * __Launch Java App__ - Debug or Run current Java project
 * __Attach to Port__ & __Attach to Process__ - Attach debugger actions. Available when __Java+ ...__ at the bottom of drop down list is selected.
     * Select this configuration, then click the ![Run](vscode/images/run.png) 
     * Select either from available process or enter the port to connect to JVM running with JDWP.
-    * __Attach to Shared Memory__ is available on Windows in addtion to above mentioned _Attach..._
+    * __Attach to Shared Memory__ is available on Windows in addition to above mentioned _Attach..._
```
Comment (sid-srini) [inline README.md:76]:
```suggestion
    * __Attach to Shared Memory__ is available on Windows in addition to the _Attach..._ configurations above.
```

**Review by sid-srini — COMMENTED** [review]
Thanks @shivam71. Please update the other full main blob URLs, such as LICENSE.txt and SECURITY.md, to their relative forms too. 
- I suppose these would also get converted to the appropriate https URLs during vsix creation by the `vsce --baseContentUrl` option. 
- Ticket microsoft/vscode#21839 shows that vsce still has no support for relative links for media in the README.

**Pull request merged**